### PR TITLE
chage broken link to proper one for jsonrpc-proxy

### PR DESCRIPTION
**Pull request by NukeManDan**
note: this link goes to code, but could also point directly to the docs instead: https://docs.rs/jsonrpc-core/ or https://substrate.dev/rustdocs/v2.0.0/jsonrpc_core/index.html

### CODE REVIEW

**Comment by NukeManDan** [comment]
I am not sure how to redirect to the most updated version on the substrate.dev/rustdocs .... any thoughts on that? like what docks.rs does...

**Comment by wheresaddie** [comment]
@Stefie is this something you can amend ?

**Comment by danforbes** [comment]
Thank you for flagging this broken link, @NukeManDan! I'm not sure if the link you've suggested is the correct one, though. Something like this may be more appropriate https://github.com/tomusdrw/jsonrpc-proxy Let me see if I can get some more info...

File: docs/tutorials/start-a-private-network/customchain.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Here are some differences from when we launched as Alice.
 - The optional `--rpc-methods=Unsafe` flag has been added. As the name indicates, this flag is not
   safe to use in a production setting, but it allows this tutorial to stay focused on the topic at
   hand. In production, you should use a
-  [JSON-RPC proxy](../../knowledgebase/getting-started/glossary#json-rpc-proxy-crate), but that
+  [JSON-RPC proxy](https://github.com/paritytech/jsonrpc), but that
   topic is out of the scope of this tutorial.
```
Comment (danforbes) [inline docs/tutorials/start-a-private-network/customchain.md:42]:
```suggestion

```

File: docs/tutorials/start-a-private-network/customchain.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Here are some differences from when we launched as Alice.
 - The optional `--rpc-methods=Unsafe` flag has been added. As the name indicates, this flag is not
   safe to use in a production setting, but it allows this tutorial to stay focused on the topic at
   hand. In production, you should use a
```
Comment (danforbes) [inline docs/tutorials/start-a-private-network/customchain.md:40]:
```suggestion
  hand.
```

**Review by danforbes — APPROVED** [review]

**Pull request merged**